app/components/ace-editor.js

```diff
@@ -0,0 +1,30 @@
+// h/t https://gist.github.com/ViktorQvarfordt/10432265
```

Comment:
It appears that the original author of this component used a two-way binding: the component is able to change data that is really owned by something else.

As an alternate pattern, ember folken are pushing "data down, actions up"- the component would fire an event handler responsible for mutating the data. See http://www.toptal.com/emberjs/the-8-most-common-ember-js-developer-mistakes#common-mistake-8-mutating-passed-in-properties-in-components

(we also use this pattern in the permissions editor component in toast/experimenter)

For an example in this application, the handler could be fired explicitly only if you click save on the text field.